src/index.html

```diff
@@ -5,8 +5,336 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <div class="menu catalog__menu">
+        <header class="header menu__header">
+          <a href="#home" class="header__logo">
+            <img src="./images/logo.png" alt="Mojo">
+          </a>
+          <nav class="header__nav nav">
+            <ul class="nav__list">
+              <li>
+                <a class="nav__link is-active" href="#">
+                  Apple
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Samsung
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Smartphones
+                </a>
+              </li>
+              <li>
+                <a
+                  class="nav__link"
+                  data-qa="nav-hover"
+                  href="#"
+                >
+                  Laptops & Computers
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Gadgets
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Tablets
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Photo
+                </a>
+              </li>
+              <li>
+                <a class="nav__link" href="#">
+                  Video
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </header>
+      </div>
+
+
+      <main class="main catalog__main">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="./images/imac.jpeg"
+            alt="imac"
+          >
+
+          <p class="card__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__rate rate">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="rate__reviews">Reviews: 5</p>
+          </div>
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-number">$2,199</p>
+          </div>
+          <a class="card__link" href="#">
+            <p class="card__link-text" data-qa="card-hover">
```

Comment:
p tag is redundant here, don't create redundant containers
you can move all styles to card__link element